test/install.mk

```diff
@@ -129,10 +129,15 @@ install-policy:
 	kubectl wait deployments istio-policy -n ${ISTIO_NS} --for=condition=available --timeout=${WAIT_TIMEOUT}
 
 # This target should only be used in situations in which the prom operator has not already been installed in a cluster.
+install-prometheus-operator: PROM_OP_NS="prometheus-operator"
 install-prometheus-operator:
-	# installs the prom operator in the default namespace
-	kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
-	kubectl wait --for=condition=available --timeout=${WAIT_TIMEOUT} deploy/prometheus-operator
+	kubectl create ns ${PROM_OP_NS} || true
+	kubectl label ns ${PROM_OP_NS} istio-injection=disabled --overwrite
+	curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed "s/namespace: default/namespace: ${PROM_OP_NS}/g" | kubectl apply -f -
```

Comment:
Can you check in the current bundle.yaml, and maybe create a kustomize.yaml file as well ? This will allow
kubectl apply -k github.com/istio/installer/kustomize/prometeus

Reply:
This was an explicit choice to avoid owning any sort of install of the prometheus operator itself. Do you believe that we should own that install?

Here it is used exclusively for testing purposes.

Reply:
Interesting point... I think we can have a copy of the installer. We can do 'curl -s..' part of the build process to update the copy. 

But we still want to be able to build a tar.gz or docker image with everything that is needed to run Istio - self-contained and hermetic. Our own operator will use it as well.

Think of it as 'vendoring' :-)

And we can add our own Kustomization on top of it.